docs/draft.md

```diff
@@ -0,0 +1,69 @@
+## Pull the Percona XtraBackup image
+
```

Comment:
No overview topic?
What is the purpose of the topics?
What will happen during the quickstart?

Reply:
Sorry, this was a draft doc, I've forgotten to remove it.